talentsearch/resources/js/components/request/CreateRequest.tsx

```diff
@@ -0,0 +1,303 @@
+              })}
+              placeholder={intl.formatMessage({
+                defaultMessage: "First and last name...",
+                description:
+                  "Placeholder for full name input in the request form.",
+              })}
+              rules={{ required: errorMessages.required }}
+            />
+            <Input
+              id="email"
+              type="email"
+              name="email"
+              label={intl.formatMessage({
+                defaultMessage: "Government e-mail",
+                description:
+                  "Label for government email input in the request form",
+              })}
+              placeholder={intl.formatMessage({
+                defaultMessage: "example@canada.ca...",
+                description:
+                  "Placeholder for government email input in the request form",
+              })}
+              rules={{ required: errorMessages.required }}
+            />
+          </div>
+          <div
+            style={{ paddingLeft: "1rem" }}
+            data-h2-flex-item="b(1of1) m(1of2)"
+          >
+            <Select
+              id="department"
+              name="department"
+              label={intl.formatMessage({
+                defaultMessage: "Department / Hiring Organization",
+                description:
+                  "Label for department select input in the request form",
+              })}
+              nullSelection={intl.formatMessage({
+                defaultMessage: "Select a department...",
+                description:
+                  "Null selection for department select input in the request form.",
+              })}
+              options={departmentOptions}
+              rules={{ required: errorMessages.required }}
+            />
+            <Input
+              id="jobTitle"
+              type="text"
+              name="jobTitle"
+              label={intl.formatMessage({
+                defaultMessage: "What is the job title for this position?",
+                description: "Label for job title input in the request form",
+              })}
+              placeholder={intl.formatMessage({
+                defaultMessage: "Developer...",
+                description:
+                  "Placeholder for job title input in the request form.",
+              })}
+              rules={{ required: errorMessages.required }}
+            />
+          </div>
+          <div data-h2-flex-item="b(1of1)">
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "In this field please include any additional details and qualifications you are seeking from the candidates such as: programming languages, certifications, knowledge, or a specific work location.",
+                description:
+                  "Blurb before additional comments textarea in the request form.",
+              })}
+            </p>
+            <TextArea
```

Comment:
Can this `textarea` be taller by default?

![Screen Shot 2021-10-29 at 14 26 45](https://user-images.githubusercontent.com/3046459/139487581-70b1926a-61ed-4a30-9da5-64415d6dd0c8.png)

Reply:
There's a button in the bottom right corner of the box that let's the user expand the size. However, it is still a bit small by default, so I'll make it bigger.